mender-convert-modify

```diff
@@ -186,10 +186,14 @@ run_and_log_cmd "echo -n ${VERSION_STRING} | sudo tee work/rootfs/etc/mender/scr
 
 log_info "Installing a custom /etc/fstab (see ${MENDER_CONVERT_LOG_FILE} for more info)"
 
-if [ "${MENDER_GRUB_EFI_INTEGRATION}" == "y" ]; then
-    boot_part_mountpoint="/boot/efi"
+if [ -n "${MENDER_BOOT_PART_MOUNT_LOCATION:-}" ]; then
```

Comment:
We have to document the new variable. We document them by:
1. Having them in `configs/mender_convert_config` with their default value and
2. In [mender-docs](https://github.com/mendersoftware/mender-docs/blob/master/04.Operating-System-updates-Debian-family/99.Variables/docs.md).

I think we should do the two, despite that as you wrote the code it is technically not needed to be defined in the default config. Reasoning: it is best to have all the developer facing info contained in the same directory.

Reply:
@lluiscampos ok tackled 1., and 2. also soon.